src/process.cpp

```diff
@@ -262,6 +283,12 @@ Process *getProcess(unsigned long inode, const char *devicename) {
   if (proc != NULL)
     return proc;
 
+  if ( !(pidsToWatch.empty()) && pidsToWatch.find(node->pid) == pidsToWatch.end() ) {    
+    if (DEBUG) 
```

Comment:
Should we enable this log on **tracemode?**

Reply:
up to you

Reply:
I will remove it, cause, it would polute the screen, and, make it harder to parse the nethogs ouput ...